### ISSUE TITLE
parameter values: opex fix in energyProduction

### ISSUE THREAD

**Issue by Piranias** [greco-project/pvcompare#13]
Hi,
I just filled in all missing parameters from the docs into our csv's. With the values as they are in the docs only the electricity grid is used. I noticed that the opex_fix is set to 965 for our pv plants, so fix operational costs (€/kW/year). Where does this high number come from? shouldn't it be zero for pv plants? Or maybe the opex_fix was mixed up with capex_fix?

eventually if i set opex_fix to zero the pv_plants and the storage are used.

**Comment by mahendrark** [comment]
Hi @Piranias 

That is supposed to be the cost for the maintenance of the PV panels. According to this [source](https://www.thumbtack.com/p/solar-panel-maintenance-cost), typical rooftop systems have to have one annual inspection (which costs 150 US $) and two cleanings of the panels. The same source mentions that there are about 15-35 panels on a typical rooftop and it costs between an average 12 - 20 USD to clean one panel. 

I will change the numbers. Taking 20 panels, 15 USD/panel cleaning cost and two cleanings per year as well as one scheduled maintenance, we arrive at 750 USD for an individual house, or ~700 €. So this is the total PV system cost. Not per kW. That was the mistake. 

There are two lingering questions. One, our system has mutliple apartment buildings. So more panels. I am not sure how to account for that. Second, opex_fix takes per kW cost. Since we do not know beforehand how big the system is going to be, what proportion of this cost (700 € or even higher if we consider the additional apartments) should we take without knowing the system size?

@SabineHaas , request your inputs.

**Comment by Piranias** [comment]
so eventually we cannot take a fix number for the opex_var because the size of our rooftop can change drastically. We cannot just assume that the area is filled with 35 panels. So it would be good to define a number for a certain rooftop size. If you say that a typical rooptop carries 35 panels, how large is the typical rooftop? I will then write a function that calculates the actual opex_var according to the actual rooptop area. 
BUT: this way we would assume that the whole area will be filled with PV plants during the simulation. Maybe we can go with this assumption first and adapt the number if the simulation result shows different. What do you think @SabineHaas ?

> There are two lingering questions. One, our system has mutliple apartment buildings. So more panels. I am not sure how to account for that. Second, opex_fix takes per kW cost. Since we do not know beforehand how big the system is going to be, what proportion of this cost (700 € or even higher if we consider the additional apartments) should we take without knowing the system size?

what is opex_fix exactly? Why is there additional costs per kW if the maintainance is already covered by opex_var?

**Comment by mahendrark** [comment]
No, `opex_var` doesn't account for the maintenance costs. `opex_var `includes costs like fuel price, etc., which doesn't concern us. 

Only `opex_fix` matters for us.` opex_fix` is the actual maintenance costs. 
I discussed with Martha earlier and she agreed with the idea that we find the average rooftop PV system size in Europe (or Germany) and divide that 700 $ / year maintenance cost discussed above for a typical rooftop PV system, by the average system size so we get that maintenance cost per kW per year, which is how the `opex_fix` is defined.

**Comment by Piranias** [comment]
ah sorry, right. I mixed it up now. Just post the typical rooftop size here when you find out

**Comment by SabineHaas** [comment]
You already discussed a lot. I just would like to add that `opex_fix` (operation and maintenance) is given in the unit EUR/kWp/year - which is also the unit we get from @mahendrark's source: dividing the costs by panels and kWp. So, we should be fine without a function sizing the costs here. (as long as the costs are dependent on kWp)

**Comment by mahendrark** [comment]
According to this [paper](https://www.mdpi.com/2313-0105/2/2/14) (page 6), the average size of a German rooftop PV system is 5 kWp. According to this [source (https://news.energysage.com/average-solar-panel-size-weight/), that system would comprise of approximately 20 panels.

So, going by the previous cost assumptions, `opex_fix` is **~140 €/kWp/year**.

**Comment by Piranias** [comment]
> You already discussed a lot. I just would like to add that `opex_fix` (operation and maintenance) is given in the unit EUR/kWp/year - which is also the unit we get from @mahendrark's source: dividing the costs by panels and kWp. So, we should be fine without a function sizing the costs here. (as long as the costs are dependent on kWp)

yes, right

**Comment by Piranias** [comment]
> According to this [paper](https://www.mdpi.com/2313-0105/2/2/14) (page 6), the average size of a German rooftop PV system is 5 kWp. According to this [source](https://news.energysage.com/average-solar-panel-size-weight/) that system would comprise of approximately 20 panels.
> 
> So, going by the previous cost assumptions, `opex_fix` is **~140 €/kWp/year**.

perfect, thanks!

**Comment by Piranias** [comment]
so after thinking again that 140 Euro/kWp still sounds pretty high I did some research again. I found (unfortunately in a _german_[ article](https://www.solaranlage-ratgeber.de/photovoltaik/photovoltaik-wirtschaftlichkeit/photovoltaik-betriebskosten)) that a cleaning only needs to be done every 1-3 years and that it costs ca 100 Euro for a up to 10 kWp system.

There is also an example calculation for a 5kWp system that sums up:
yearly check: 150 Euro 
cleaning (1-3 years): 100 Euro
Insurance: 50 Euro

that makes 250 Euro pear year and **50 Euro/kWp**

do we take insurance into account?
